terra/logger.py

```diff
@@ -221,16 +351,55 @@ def setup_logging_ipython_exception_hook(self):
       original_exception = InteractiveShell.showtraceback
 
       def handle_traceback(*args, **kwargs):  # pragma: no cover
-        getLogger(__name__).error("Uncaught exception",
-                                  exc_info=sys.exc_info())
+        try:
+          getLogger(__name__).critical("Uncaught exception",
```

Comment:
# Use getLogger instead of logger (defined below) incase there is an
        # exception on import, this will make it easier to get a normal error
        # message